torchsparse/nn/modules/bev.py

```diff
@@ -86,12 +86,11 @@ def forward(self, input: SparseTensor) -> torch.Tensor:
         coords, feats, stride = input.coords, input.feats, input.stride
```

Comment:
@Xiuyu-Li, please revert these changes. These are required for proper backward support.